golang/pkg/crane/k8s/deployment.go

```diff
@@ -192,6 +192,22 @@ func (d *Deployment) Scale(namespace, name string, target int) error {
 	return err
 }
 
+func (d *Deployment) GetPods(namespace, name string) ([]coreV1.Pod, error) {
```

Comment:
Tests are missing 🦅

Reply:
I'm not really sure how I'd go about testing this. I added a test yaml with some deployments and the test can check for them, but I think integration tests are not running in the pipeline.